dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -62,6 +64,124 @@ protected WorkflowResource(final WorkflowHelper workflowHelper,
         this.workflowAPI    = workflowAPI;
     }
 
+    /**
+     * Returns all schemes non-archived. 401 if the user does not have permission.
+     * @param request  HttpServletRequest
+     * @return Response
+     */
+    @GET
+    @Path("/schemes")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response findSchemes(@Context final HttpServletRequest request) {
+
+        final InitDataObject initDataObject = this.webResource.init
+                (null, true, request, true, null);
+        Response response;
+        List<WorkflowScheme> schemes;
+
+        try {
+
+            Logger.debug(this, "Getting the workflow schemes");
+            schemes   = this.workflowHelper.findSchemes();
+
+            response  =
+                    Response.ok(new ResponseEntityView(schemes)).build(); // 200
+        } catch (Exception e) {
+
+            Logger.error(this.getClass(),
+                    "Exception on findSchemes exception message: " + e.getMessage(), e);
+            response = (e.getCause() instanceof SecurityException)?
+                    ExceptionMapperUtil.createResponse(e, Response.Status.UNAUTHORIZED) :
+                    ExceptionMapperUtil.createResponse(e, Response.Status.INTERNAL_SERVER_ERROR);
+        }
+
+        return response;
+    } // findSchemes.
+
+    /**
+     * Returns all schemes for the content type. 401 if the user does not have permission.
+     * @param request  HttpServletRequest
+     * @return Response
+     */
+    @GET
+    @Path("/schemes/contenttypes/{contentTypeId}")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response findSchemesByContentType(@Context                         final HttpServletRequest request,
+                                                   @PathParam("contentTypeId")      final String contentTypeId) {
+
+        final InitDataObject initDataObject = this.webResource.init
+                (null, true, request, true, null);
+        Response response;
+        List<WorkflowScheme> schemes;
+        Map<String, Object>  resultMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "resultMap" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

Reply:
Done